### PR TITLE
fix(ui): prevent already signed-up user from signing up again

### PR DESCRIPTION
**Pull request by ghost**
## Description
* prevent already signed up user from signin up again
* show message and link to sign-in on step four

## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: sdk/src/auth/index.ts [hunk 1]
```diff
@@ -133,12 +133,11 @@ export default class AWF_Auth implements AWF_IAuth {
...
     );
     return check.pipe(
-      tap(response => {
-        if (!response.data || response.data?.errors) {
-          const err = new Error('This ethereum key is not registered');
-          err.name = 'UserNotRegistered';
-          throw err;
-        }
+      tap(() => {
+        this._globalChannel.next({
```
Comment (kenshyx) [inline sdk/src/auth/index.ts]: if this replaces the error throw case then it should still keep the guard rule

Comment (ghost) [reply sdk/src/auth/index.ts]: I can add it, but from what ive seen it never threw here, getProfile has its own throw which always happened before this

Comment (kenshyx) [reply sdk/src/auth/index.ts]: yes, but this is part of the `checkIfSignedUp` method here

Comment (kenshyx) [reply sdk/src/auth/index.ts]: I still don't understand the use-case for the event emitted here

Comment (kenshyx) [reply sdk/src/auth/index.ts]: this event will be emitted on every `checkIfSignedUp` method call

Comment (ghost) [reply sdk/src/auth/index.ts]: Hmmm good point. I used it to keep displaying the loader for connect address, until we know whether to stop the sign-up or not (already registered). But I guess this is something we can live without.

Comment (kenshyx) [reply sdk/src/auth/index.ts]: you can still emit it but it needs to be aware of the response, for example you could also attach some data to the event like `isRegistered`

Comment (ghost) [reply sdk/src/auth/index.ts]: Turns out there was no actual need for it in the end 🤦‍♂️🤦‍♂️ I'll let myself out

**Pull request merged**